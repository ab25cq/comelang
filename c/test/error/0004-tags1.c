/*
PATTERN: redefinition
PATTERN: c:8:
*/

struct X;
struct X {};
struct X {int x;};
