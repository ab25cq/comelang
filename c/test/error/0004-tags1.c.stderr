redefinition of tag X at test/error/0004-tags1.c:8:1
struct X {int x;};
^
